Seminar02/solutions/Practicum 2 C++ Files/Ex3.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    double a,b; //It is not said that the number is whole in the problem
+    cin >> a >> b;
+
+    if(a==b){
```

Comment:
use very small number like 0,00001 to avoid the computer error when comparing doubles, 
like that : if((a-b)<0,0001), also you can use the ternary operator, again I saw that you know how